src/Swoole/SwooleClient.php

```diff
@@ -150,23 +151,37 @@ public function sendResponseHeaders(Response $response, SwooleResponse $swooleRe
     /**
      * Send the headers from the Illuminate response to the Swoole response.
      *
-     * @param  \Symfony\Component\HtpFoundation\Response  $response
+     * @param  \Laravel\Octane\OctaneResponse  $response
      * @param  \Swoole\Http\Response  $response
      * @return void
      */
-    protected function sendResponseContent(Response $response, SwooleResponse $swooleResponse): void
+    protected function sendResponseContent(OctaneResponse $octaneResponse, SwooleResponse $swooleResponse): void
     {
-        if ($response instanceof StreamedResponse && property_exists($response, 'output')) {
-            $swooleResponse->end($response->output);
+        if ($octaneResponse->response instanceof StreamedResponse && property_exists($octaneResponse->response, 'output')) {
```

Comment:
I think this case is obsolete and never used? The output property is never set in Octane, as discussed in https://github.com/laravel/octane/issues/109